project/src/main/webapp/components/user-home/docs-component.js

```diff
@@ -3,37 +3,39 @@ import {html, LitElement} from 'https://unpkg.com/@polymer/lit-element/lit-eleme
 export class DocsComponent extends LitElement {
   static get properties() {
     return {
-      documents: {type: Array},
+      documents: {type: Object},
     };
   }
 
   constructor() {
     super();
-    this.documents = [
-      {'title': 'A doc', 'timestamp': 'June 16, 5:28PM'},
-      {'title': 'Another doc', 'timestamp': 'June 15, 1:30PM'}
-    ]
+    this.documents = [];
   }
 
   // Remove shadow DOM so styles are inherited
   createRenderRoot() {
     return this;
   }
 
+  getServletData() {
+    /*fetch('/UserHome').then((response) => response.json()).then((documentList) => {
+      this.documents = documentList;
+    });*/
+    console.log("Rewrite when UserHomeServlet can return data.");
```

Comment:
I'm guessing this is the next step as soon as we iron out the problem with adding a document to a user?